src/univariate/continuous/biweight.jl

```diff
@@ -54,16 +54,16 @@ end
 
 @quantile_newton Biweight
 
-function mgf(d::Biweight{T}, t::Real) where T<:Real
-    a = d.σ*t
+function mgf(d::Biweight, t::Number)
+    a = d.σ * t
     a2 = a^2
-    a == 0 ? one(T) :
+    iszero(a) ? one(Base.promote_typeof(a, d.μ)) :
     15exp(d.μ * t) * (-3cosh(a) + (a + 3/a) * sinh(a)) / (a2^2)
```

Comment:
So my suggestion would be to just use
```suggestion
    result = 15 * exp(d.μ * t) * (-3 * cosh(a) + (a + 3/a) * sinh(a)) / (a2^2)
    return iszero(a) ? one(result) : result
```

Reply:
That's not type stable either no? If `a2 == 0` then you might get some `Inf` or `NaN`

Reply:
This will always be type stable (and is therefore the common approach in Distributions): it returns either `result` or `one(result)` (in the case `iszero(a)` or equivalently `iszero(a2)`), which are both of type `typeof(result)` (we don't work with unitful numbers, `oneunit(result)` would be even more correct).